check_for_update.go

```diff
@@ -57,30 +57,26 @@ func getLatestVersion() (*semver.Version, error) {
 	return version, nil
 }
 
-func getLatestVersionFromCache() (version *semver.Version, modified time.Time, ok bool) {
+func getLatestVersionFromCache() (version *semver.Version, modified time.Time, err error) {
 	f, err := os.Open(versionCacheFilename)
 	if err != nil {
-		ok = false
-		return
+		return nil, time.Now(), err
 	}
 	defer f.Close()
 
 	stat, err := f.Stat()
 	if err != nil {
-		ok = false
-		return
+		return nil, time.Now(), err
 	}
 	modified = stat.ModTime()
 
 	content, err := ioutil.ReadAll(f)
 	if err != nil {
-		ok = false
-		return
+		return nil, time.Now(), err
```

Comment:
fixed in https://github.com/phrase/phraseapp-client/pull/68/commits/39738d927ce2cac252ee8e3abd203446cab08fcd